.editorconfig

```diff
@@ -0,0 +1,283 @@
+# See https://github.com/RehanSaeed/EditorConfig for updates to this file.
+# See http://EditorConfig.org for more information about .editorconfig files.
+
+#################
+# Common Settings
+#################
+
+# This file is the top-most EditorConfig file
+root = true
+
+# All Files
+[*]
+charset = utf-8
+end_of_line = crlf
+indent_style = space
+indent_size = 4
+insert_final_newline = false
```

Comment:
`false`? really..? really..!?

Reply:
Agree, I would prefer **insert_final_newline = true** and **end_of_line = lf**

Reply:
Why use one type of setting over another in this case?

Reply:
https://stackoverflow.com/questions/2287967/why-is-it-recommended-to-have-empty-line-in-the-end-of-file

These are the reasons?

Reply:
Basically yes. But mostly it's just a cosmetic thing for me. I'm so used to seeing new lines at the end of a file, so when it's not there, it looks messy and compact.

Note that the editor config for roslyn (https://github.com/dotnet/roslyn/blob/master/.editorconfig) that you linked to in the related issue sets `insert_final_newline = true`. It's a very common thing to do.